pages/en/hosted-service/migrating-subgraph.mdx

```diff
@@ -4,11 +4,18 @@ title: Migrating an Existing Subgraph to The Graph Network
 
 ## Introduction
 
-This is a guide for the migration of subgraphs from the Hosted Service (also known as the Hosted Service) to The Graph Network. The migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
+This is a guide on how to migrate your subgraph from the Hosted Service to The Graph Network. The migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
 
-This will tell you everything you need to know about how to migrate to the decentralized network and manage your subgraphs moving forward. The process is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
+The process of migration is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
 
-### Migrating An Existing Subgraph to The Graph Network
+### When not to Migrate?
+
+If your subgraph:
+
+- Is not generating too many queries, and at the same time you are not willing to self-signal. 
```

Comment:
Good idea with this section. We should be careful about the wording here. Too little queries is not a reason to not migrate. Also self-signalling is not necessary a problem. But these are good reasons to not migrate:
- Usage of IPFS
- Usage of full-text search
- Deployed on other chains than Ethereum mainnet